docs/resources/organizations_policy_attach.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Organizations"
+---
+
+# huaweicloud_organizations_policy_attach
+
+Manages an Organizations policy attach resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable policy_id {}
+variable entity_id {}
+
+resource "huaweicloud_organizations_policy_attach" "test"{
+  policy_id = var.policy_id
+  entity_id = var.entity_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `policy_id` - (Required, String, ForceNew) Specifies the ID of the policy.
+
+  Changing this parameter will create a new resource.
+
+* `entity_id` - (Required, String, ForceNew) Specifies the unique ID of the root, OU, or account.
+
+  Changing this parameter will create a new resource.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID which is formatted `<policy_id>/<entity_id>`.
+
+* `name` - Indicates the name of the entity.
+
+* `type` - Indicates the type of the entity.
```

Comment:
how about use entity_name and entity_type

Reply:
done